src/course/en/course.json

```
@@ -116,6 +116,18 @@
             }
         ]
     },
+    "_start": {
+        "_isEnabled": false,
+        "_startIds": [
+            {
+                "_id": "co-05",
+                "_skipIfComplete": false,
```

I think this might be the more common use-case?
```suggestion
                "_skipIfComplete": true,
```